src/Form/test-cases/itemfix_signup.html

```diff
@@ -0,0 +1,76 @@
+<!-- https://www.itemfix.com/user?a=register -->
+<form>
+    <div class="col-sm-12 pad-none add-new-item-outer contact-form-outer">
+
+        <div class="col-sm-12 pad-none add-item-license-con">
+            <div class="col-sm-12 pad-none form-group add-item-license-form">
+                <label>User Name</label>
+                <input type="text" name="name" class="form-control" data-manual-scoring="username">
+            </div>
+
+            <div class="col-sm-12 pad-none form-group add-item-license-form">
+                <label>Email</label>
+                <input type="text" name="email1" class="form-control" data-ddg-autofill="true"
+                    data-manual-scoring="emailAddress">
+            </div>
+
+            <div class="col-sm-12 pad-none form-group add-item-license-form">
+                <label>Repeat Email</label>
+                <input type="text" name="email2" class="form-control" data-ddg-autofill="true"
+                    data-manual-scoring="emailAddress">
+            </div>
```

Comment:
What type should the 2nd field here have?

Reply:
Repeat email fields have `emailAddress` (we fill them both with the same autofill event).